### PR TITLE
Replaced "en orden de" with "Para poder"

### PR DESCRIPTION
**Pull request by pcp16**
"En orden de" sounds awkward in Spanish, seems like a literal translation of "In order to". Changed to "para poder".

### CODE REVIEW

**Comment by pcp16** [comment]
Added another commit with some typos corrected.

**Comment by pcp16** [comment]
Hope I'm doing this right ...

**Comment by das-g** [comment]
@DjangoGirls/es-pr please review.

**Comment by das-g** [comment]
ping @DjangoGirls/es-pr

File: es/domain/README.md [hunk 1]
```diff
@@ -32,36 +32,36 @@ Y llena la siguiente información
...
-* Copia la **ruta a tu virtualenv** y colocala en un lugar seguro.
-* Da click a través de tu **archivo de configuración wsgi**, copia el contenido y pegalo en un lugar seguro.
+* Copia la **ruta a tu virtualenv** y colócala en un lugar seguro.
+* Haz click a través de tu **archivo de configuración wsgi**, copia el contenido y pégalo en un lugar seguro.
 
-Siguiente, **elimina** tu vieja aplicación. No te preocupes, esto no elimina nada de tu código, solamente apaga el dominio en `nombredeusuario.pythonanywhere.com`. Siguiente, crea una nueva aplicación y sigue estos pasos:
+Seguidamente, **elimina** tu vieja aplicación. No te preocupes, esto no elimina nada de tu código, solamente apaga el dominio en `nombredeusuario.pythonanywhere.com`. Después, crea una nueva aplicación y sigue estos pasos:
 
-* Ingresa tu nombre de dominio
+* Escrine el nombre de tu dominio
```
Comment (adRn-s) [inline es/domain/README.md:54]: `s/Escrine/Escribe/`

**Review by adRn-s — CHANGES_REQUESTED** [review]
Other than the mentioned typo, this looks good to merge!

File: es/domain/README.md (hunk 1 above)
Comment (das-g) [reply es/domain/README.md:54]: Thus
```suggestion
* Escribe el nombre de tu dominio
```

**Comment by das-g** [comment]
Thank you, @pcp16 and @adRn-s, and sorry for the long wait.

**Pull request merged**